.github/workflows/ubuntu.yml

```diff
@@ -27,8 +27,8 @@ jobs:
     env:
       CC: ${{ matrix.compiler-version.cc }}
       CXX: ${{ matrix.compiler-version.cxx }}
-      CPLEX_URL: ${{ secrets.CPLEX_URL }}
-      SOPLEX_URL: ${{ secrets.SOPLEX_URL }}
+      CPLEX_URL: ${{ secrets.CPLEX129_LINUX_URL }}
+      SOPLEX_URL: ${{ secrets.SOPLEX311_URL }}
```

Comment:
I added 'LINUX' to the CPLEX key, because we have now the same key for Windows too (but with 'WINDOWS'). For SoPlex, I do not expect this necessary, as we are building from sources (once that works).

I added the version numbers to the secret. At least for  Windows changing the CPLEX version is more than changing the secret to another URL.